scripts/utils/spawn.ts

```diff
@@ -0,0 +1,28 @@
+import childProcess from 'child_process';
+
+/**
+ * Utility to do spawn but as a Promise
+ * @param command
+ * @param args
+ * @param options
+ */
+export const spawn = (command: string, args: string[], options: childProcess.SpawnOptions = {}) =>
+    new Promise<void>((resolve, reject) =>
+    {
+        const child = childProcess.spawn(command, args, {
+            stdio: 'inherit',
+            cwd: process.cwd(),
+            // See https://nodejs.org/api/child_process.html#spawning-bat-and-cmd-files-on-windows
+            shell: process.platform.startsWith('win'),
```

Comment:
```suggestion
            shell: process.platform === 'win32',
```

We don't need `startsWith` here, since 64-bit Windows is still called `'win32'`.